src/extensions/interpolation.jl

```diff
@@ -164,6 +167,27 @@ end
 # Writers
 #
 
+# JuliaExpression
+
+function write_html(jv::JuliaExpression, rend, node, enter)
+    tag(rend, "span", attributes(rend, node, ["class" => "julia-expr"]))
+    print(rend.buffer, sprint(print, '$', "($(jv.ex))"))
+    tag(rend, "/span")
+end
+
+function write_latex(jv::JuliaExpression, rend, node, enter)
+    print(rend.buffer, sprint(print, '\\', '$', "($(jv.ex))"))
+end
```

Comment:
Actually I just realized the LaTeX is problematic, since it would get interpreted as valid LaTeX commands..

Reply:
Ah, maybe just escape the printed expression. Pretty sure there's some latex escaping code somewhere in the package...

Reply:
Fixed it to use `latex_escape`, as is done for `Code`.